admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/page/domain/PageImpl.java

```diff
@@ -74,19 +75,59 @@
 @Inheritance(strategy = InheritanceType.JOINED)
 @Table(name = "BLC_PAGE")
 @EntityListeners(value = { AdminAuditableListener.class })
-@AdminPresentationOverrides(
-    {
-        @AdminPresentationOverride(name="auditable.createdBy.id", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="auditable.updatedBy.id", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="auditable.createdBy.name", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="auditable.updatedBy.name", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="auditable.dateCreated", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="auditable.dateUpdated", value=@AdminPresentation(readOnly = true, visibility = VisibilityEnum.HIDDEN_ALL)),
-        @AdminPresentationOverride(name="pageTemplate.templateDescription", value=@AdminPresentation(excluded = true)),
-        @AdminPresentationOverride(name="pageTemplate.templateName", value=@AdminPresentation(excluded = true)),
-        @AdminPresentationOverride(name="pageTemplate.locale", value=@AdminPresentation(excluded = true))
-    }
-)
+@AdminPresentationMergeOverrides(value = { 
+    @AdminPresentationMergeOverride(name = "auditable.createdBy.id",
+        mergeEntries = { 
+            @AdminPresentationMergeEntry(propertyType = PropertyType.AdminPresentation.READONLY, booleanOverrideValue = true),
+            @AdminPresentationMergeEntry(propertyType = PropertyType.AdminPresentation.VISIBILITY, overrideValue = "HIDDEN_ALL")
```

Comment:
These literal "HIDDEN_ALL" should probably use VisibilityEnum.HIDDEN_ALL

Reply:
Required type for `overrideValue` is String so it does not accept enum.
![image](https://user-images.githubusercontent.com/35038090/172855459-a3f357af-7344-4170-af27-b99c9ebddf14.png)

Reply:
We can introduce `overrideVisibility` and update everywhere. But that will have to be separate ticket. This is how its set right now in `BasicFieldMetadataProvider`
```
} else if (entry.getKey().equals(PropertyType.AdminPresentation.VISIBILITY)) {
      fieldMetadataOverride.setVisibility(VisibilityEnum.valueOf(stringValue));
```